resource/labeling-app/components/text-sample/textArea.js

```diff
@@ -125,18 +104,16 @@ const TextArea = {
             captionSpan.style.color = colorStrOpaque
             selectionWrapper.appendChild(captionSpan);
 
-            selected && selectionWrapper.scrollIntoView();
+            selected && selectionWrapper.scrollIntoView({block: "nearest", behavior: "smooth", });
         },
-        getLabeledText(start, end, tokenStart, tokenEnd) {
+        getLabeledText(start, end) {
             return {
                 label: this.selectedLabel,
                 text: this.getTextFromBoundaries(start, end),
                 start: start,
                 end: end,
-                tokenStart: tokenStart,
-                tokenEnd: tokenEnd,
                 draft: false,
-                selected: !this.selectedLabel,
+                selected: true,//!this.selectedLabel,
```

Comment:
why is selected always true here?

Reply:
O changed it because of what we discussed about : After marking a token, it should be automatically selected